syntax = "proto3";

package Configs;

message Weather {
  double ambientTemperature = 1;
  double trackTemperature = 2;
  double humidity = 3;
}

message SessionConfig {
  string trackLocation = 1;
  string trackLayout = 2;
  string sessionName = 3;
  string driver = 4;
  string date = 5;
  string time = 6;
  Weather weather = 7;
  string notes = 8;
  double canlibVersion = 9;
  uint64 startTimestamp = 10;
  uint64 endTimestamp = 11;
}
